components/LinkButton.js

```diff
@@ -0,0 +1,29 @@
+import Link from "next/link";
+import styled from "styled-components";
+
+export default function LinkButton({ href, name, fullWidth }) {
+  return (
+    <StyledLink fullwidth={fullwidth} href={href}>
+      {name}
```

Comment:
```suggestion
export default function LinkButton({ href, children, fullWidth }) {
  return (
    <StyledLink fullwidth={fullwidth} href={href}>
      {children}
```
Same explanation as for the `Headline` component